src/main/java/org/apache/commons/configuration2/AbstractConfiguration.java

```diff
@@ -119,7 +119,7 @@ public abstract class AbstractConfiguration extends BaseEventSource implements C
     /**
      * Creates a new instance of {@code AbstractConfiguration}.
      */
-    public AbstractConfiguration() {
+    protected AbstractConfiguration() {
```

Comment:
1) Don't break binary compatibility
2) This is not what you say this PR is about.